### DISCUSSION TITLE
Другой выбор игрока для блока чата

### DISCUSSION THREAD

**Discussion by gat3v** [Sokol-Games-GmbH/PixelCombats.PublicApi#137] [Ideas]
**Бывают случаи когда игроки с другим шрифтом в никнейме заходят в сервер не заходя в команду и спамят чем угодно.**

Сделайте лучше чтоб игрок не вводил никнейм игрока для блокировки чата ему, а лучше пусть сервер будет выводить всех игроков на сервере в список, и при нажатие на игрока блок чата, и повторное нажатие на игрока разблок чата. Можете конечно подправить идею или же изменить.

**Comment by kkohno** [comment]
понял - многие жалуются на это.. есть такое. поправим

**Reply by gat3v** [reply]
Ок, было бы хорошо с новым выбором
